internal/updater/cmd-updater.go

```diff
@@ -35,6 +35,7 @@ type CmdUpdater struct {
 	Timeout              time.Duration
 	EnableCI             bool
 	PackageLockFile      string
+	ForceUpdateLock      bool
```

Comment:
Same, to make this clear to the user, use something like `EnablePerPackageUpdatePause`